process/transactions/esdtTransactionProcessor.go

```diff
@@ -178,66 +178,62 @@ func (etp *esdtTransactionProcessor) searchSCRWithNonceOfNFTAndPutInAlteredAddre
 			IsESDTOperation: isESDTNotNFT,
 			IsNFTOperation:  isNFT,
 			TokenIdentifier: tokenIdentifier,
-			NFTNonceString:  nonceStr,
+			NFTNonce:        nonceStr,
 		})
 		return true
 	}
 
 	return
 }
 
-func (etp *esdtTransactionProcessor) extractNonceString(scr *data.ScResult) string {
+func (etp *esdtTransactionProcessor) extractNonceString(scr *data.ScResult) uint64 {
```

Comment:
rename the function. drop the `String` suffix

Reply:
done.